2.4/test/run

```diff
@@ -280,9 +280,53 @@ function run_change_password_test() {
     echo "  Success!"
 }
 
+function run_mount_config_test() {
+    local name="mount_config"
+    echo "  Testing config file mount"
+    local database='db'
+    local user='user'
+    local password='password'
+    local admin_password='adminPassword'
+
+    local volume_dir
+    volume_dir=`mktemp -d --tmpdir mongodb-testdata.XXXXX`
+    config_file=$volume_dir/mongod.conf
+    echo """dbpath=/var/lib/mongodb/dbpath
```

Comment:
Why the triple double quotes here?!

Looks like Python, but this is not Python :P